spikeextractors/extractors/klustaextractors/klustaextractors.py

```diff
@@ -37,13 +49,16 @@ def __init__(self, folder_path):
         self._kwargs = {'folder_path': str(Path(folder_path).absolute())}
 
 
+# noinspection SpellCheckingInspection
 class KlustaSortingExtractor(SortingExtractor):
     extractor_name = 'KlustaSortingExtractor'
     installed = HAVE_KLSX  # check at class level if installed or not
     installation_mesg = "To use the KlustaSortingExtractor install h5py: \n\n pip install h5py\n\n"  # error message when not installed
     is_writable = True
     mode = 'file_or_folder'
 
+    default_cluster_groups = {0: 'Noise', 1: 'MUA', 2: 'Good', 3: 'Unsorted'}
```

Comment:
this is fixed in the kwik file, right? 

Then I recommend only saving the `cluster_groups_name` and not the `.cluster_groups_int`. Do you agree?

Reply:
No these are the gui's defaults. I think you can also add manual labels, but I/our lab doesn't use this feature, so I don't know how it would look like. I would expect a new group integer (ie '4'), which stands for the custom label.

With that context, how would you approach this?

Reply:
I've added this:
`                assert cluster_group in self.default_cluster_groups.keys(), f'cluster_group not in "default_dict: {cluster_group}'`

if that will ever fail for someone, we can pick it up from there

Reply:
That could work